config/mail.global.php.dist

```diff
@@ -2,20 +2,20 @@
 
 declare(strict_types=1);
 
+use Symfony\Component\Mailer\Transport\SendmailTransport;
```

Comment:
do we want to have 2 places in which to change the transport ?
sendmail versus SMTP